src/fr-policy/enums.ts

```diff
@@ -0,0 +1,111 @@
+import { PolicyParams } from 'auth/interfaces';
```

Comment:
Make sure the import path is relative to the current file, otherwise the project importing the SDK will not be able to import the module.